qemu/tests/qsd_qmp_cmd_check.py

```diff
@@ -0,0 +1,92 @@
+"""QSD QMP commands test"""
+import json
+
+from provider.qsd import QsdDaemonDev
+from virttest import error_context
+from virttest.qemu_monitor import QMPCmdError
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test QSD QMP commands .
+    Steps:
+        1) Run QSD with one export vhost-user-blk.
+        2) Check the export id existence.
+        3) Delete relevant object,blockdev nodes and export
+        3) Add relevant object,blockdev nodes and export
+        3) Delete relevant object,blockdev nodes and export
+    """
+
+    logger = test.log
+    qsd = None
+
+    try:
+        qsd_name = params["qsd_namespaces"]
+        qsd = QsdDaemonDev(qsd_name, params)
+        qsd.start_daemon()
+        img = params["qsd_images_qsd1"]
+        monitor = qsd.monitor
+
+        img_attrs = qsd.images[img]
+        obj_iothread = json.loads(params["obj_iothread"])
+        obj_throttle = json.loads(params["obj_throttle"])
+        prot_opts = img_attrs["protocol"]
+        fmt_opts = img_attrs["format"]
+        export_opts = img_attrs["export"]
+        logger.debug("Check the export list")
+        out = monitor.cmd("query-block-exports")
+        test.assertTrue(out[0]["id"] == export_opts["id"],
+                        "Can not find export")
+
+        logger.debug("Delete the export,blockdev and object")
+        monitor.block_export_del(export_opts["id"])
```

Comment:
Because the relevant object, blockdev nodes and export will be deleted twice, I suggest writing a function for being called. Thanks.